src/main/java/com/btkelly/gnag/models/ViolationComment.java

```diff
@@ -25,7 +25,7 @@
 
     public ViolationComment(boolean failBuild, String commentMessage) {
```

Comment:
Well, you don't have the Android support ones, so I guess you'd have to be using IntelliJ's?